Applications/CLI/Tests.cmake

```diff
@@ -520,6 +520,52 @@ if(NOT OGS_USE_MPI)
     #     EXECUTABLE_ARGS tes-inert-wedge.prj
     # )
 
+    # Liquid flow
+    AddTest(
+        NAME LiquidFlowProcess_sat1D
```

Comment:
Is it possible to name the tests, s.t. its general setup is clear from the name.
Otherwise I've to look into the prj files every time to find out what that could be.
Something like LiquidFlow_1D_line100_saturationZone_quadraticPressureLeft?